README.rst

```diff
@@ -80,24 +80,25 @@ been used to create the diagram.
 
 Installation
 ------------
-We highly recommend to use anaconda_ and install straditize::
+We highly recommend to use anaconda_ and install straditize into an own environment::
```

Comment:
I'd change `an own` to `its own`

Reply:
thanks! resolved by https://github.com/Chilipp/straditize/pull/6/commits/5522234bca9dbad38072e85769addc84145a7096